region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -134,6 +135,20 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Moondance Clip",
+      "notable": false,
+      "entranceCondition": {
+        "comeInWithMoondance": {}
+      },
+      "requires": [
+        "canMoondance",
+        "Plasma",
+        "Wave"
+      ],
+      "clearsObstacles": ["A"]
```

Comment:
```suggestion
      "clearsObstacles": ["A"],
      "devNote": "Coming in with blue speed would be an alternative way to get past the hoppers, but this may be tricky to model."
```